src/main/scala/io/iohk/ethereum/jsonrpc/JsonRpcController.scala

```diff
@@ -182,7 +195,10 @@ class JsonRpcController(
             case Right(success) => successResponse(rpcReq, success)
             case Left(error) => errorResponse(rpcReq, error)
           }
-          .recover { case ex => errorResponse(rpcReq, InternalError) }
+          .recover { case ex =>
+            log.error("Failed to handle RPC request", ex)
```

Comment:
👍 
We generally miss logging in this area. Would be really helpful to know which methods are being called from Ethereum Wallet.